src/components/change-view.jsx

```diff
@@ -173,16 +211,67 @@ export default class ChangeView extends React.Component {
         this.setState(previousState => ({collapsedView: !previousState.collapsedView}));
     }
 
+    _markAsSignificant (event) {
+      event.preventDefault();
+      if (isDisabled(event.currentTarget)) return;
+
+      let annotation = this.state.annotation;
+      if (!annotation.significance || annotation.significance < 0.5) {
+        annotation = Object.assign({}, annotation, {significance: 0.5});
+        this._updateAnnotation(annotation);
+        this._saveAnnotation(annotation);
+
+        this.setState({addingToImportant: true});
+        const onComplete = () => this.setState({addingToImportant: false});
+
+        this.context.localApi.addChangeToImportant(
+          this.props.page,
+          this.state.a,
+          this.state.b,
+          this.state.annotation
+        )
+          .then(onComplete, onComplete);
+      }
+    }
+
+    _addToDictionary (event) {
+      event.preventDefault();
+      if (isDisabled(event.currentTarget)) return;
+
+      let annotation = this.state.annotation;
+      if (!annotation.isDictionary) {
+        annotation = Object.assign({}, annotation, {isDictionary: true});
+        this._updateAnnotation(annotation);
+        this._saveAnnotation(annotation);
+
+        this.setState({addingToDictionary: true});
+        const onComplete = () => this.setState({addingToDictionary: false});
+
+        this.context.localApi.addChangeToDictionary(
+          this.props.page,
+          this.state.a,
+          this.state.b,
+          this.state.annotation
```

Comment:
I think I probably already had this written this way before I had the setup with `annotation` and failed to change it. Practically speaking, there’s no difference (because the sheet does not use the values for `significance` or `isDictionary`), but you’re right, it would be more correct to use `annotation` here.